setup/windows/README.md

```diff
@@ -49,6 +49,6 @@ $ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml -vv
 ```
 
 The servers should logon automatically at boot and start the Jenkins slave, but there is no automatic restart procedure for crashes (for now).
-Hence, to start Jenkins after running `ansible-playbook`, the server must be restarted or the slave must be manually started with RDP.
```

Comment:
Yes there is a command for doing this (https://github.com/nodejs/build/blob/master/setup/windows/vs2015-ansible-playbook.yaml#L29). Perhaps we could add another reboot after `common-ansible-playbook.yaml` has been played?